apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore.IntegrationTests/Logging/LoggingTest.cs

```diff
@@ -501,9 +501,9 @@ public string FormatParameters(string id)
 
         public string ScopeFormatParameters(string id)
         {
-            using (_logger.BeginScope(nameof(ScopeFormatParameters) + " - {id}"))
+            using (_logger.BeginScope(nameof(ScopeFormatParameters) + " - {id}", id))
             {
-                string message = GetMessage(nameof(Scope), id);
+                string message = GetMessage(nameof(ScopeFormatParameters), id);
```

Comment:
We use the name of the method as part of the message to more easily to which test belongs each of the log entries.